src/Components/CollectionsHubRails/OtherCollectionsRail/OtherCollectionEntity.tsx

```diff
@@ -33,8 +35,21 @@ export const OtherCollectionEntity: React.FC<CollectionProps> = ({
   member,
 }) => {
   const { slug, thumbnail, title } = member
+  const { trackEvent } = useTracking()
+
+  const onClickLink = () => {
+    trackEvent({
+      action_type: Schema.ActionType.Click,
+      context_page: Schema.PageName.CollectionPage,
+      context_module: Schema.ContextModule.OtherCollectionsRail,
+      context_page_owner_type: Schema.OwnerType.Collection,
+      type: Schema.Type.Link,
+      destination_path: `${sd.APP_URL}/collection/${slug}`,
+    })
+  }
+
   return (
-    <StyledLink href={`${sd.APP_URL}/collection/${slug}`}>
+    <StyledLink href={`${sd.APP_URL}/collection/${slug}`} onClick={onClickLink}>
```

Comment:
#superminor, it's common to internally (as in locally within a file) refer to `onSomething` handlers  as `handleSomething`; `onSomething` generally refers to functions being passed down as props, from a different scope.